eos_data_distribution/ndn/command.py

```diff
@@ -70,22 +70,39 @@ def makeInterest(cmd, local=True, controlParameters={},
 def main():
     from gi.repository import GLib
     from . import base
-    import argparse
+    from tests import utils
 
     logging.basicConfig(level=logging.DEBUG)
 
-    parser = argparse.ArgumentParser(description='Command Interest Tests')
-    parser.add_argument("-n", "--name")
+    parser = utils.process_args(description='Command Interest Tests')
```

Comment:
General comment about this commit (“ndn: prepare commands to be testable”): I have no idea what is going on here. What are your plans for the testing, and why aren’t they discussed in the commit message? What do all these arguments do, and why don’t they have `help` to explain it?